tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -22,8 +23,12 @@
 from tests.sentry.api.endpoints.test_organization_metrics import MOCKED_DERIVED_METRICS
 
 
-def _indexer_record(org_id: int, string: str) -> int:
-    return indexer.record(use_case_id=UseCaseKey.RELEASE_HEALTH, org_id=org_id, string=string)
+def indexer_record(use_case_id: UseCaseKey, org_id: int, string: str) -> int:
+    return indexer.record(use_case_id=use_case_id, org_id=org_id, string=string)
+
+
+perf_indexer_record = partial(indexer_record, UseCaseKey.PERFORMANCE)
```

Comment:
So if we make the metrics API only work on releasehealth, don't we always want to go to that usecase in this entire file?

Reply:
correct, that was my assumption too, when @ahmedetefy suggested that this API endpoint is RELEASE_HEALTH only.
https://github.com/getsentry/sentry/blob/a2e7352dd2257e52cd3b5596d53f85932ef84ea3/src/sentry/api/endpoints/organization_metrics.py#L104-L143
Otherwise we need to expose it to query parameter.

Reply:
i think either way makes sense. i'll leave the decision up to you. i don't think mixing the usecases in tests like a ton of sense, but since it's just tests and it all goes to the same DB in CI i guess it doesn't matter